server/src/server.ts

```diff
@@ -640,7 +612,6 @@ function initializePolisHelpers() {
     tid?: any,
     voteType?: any,
     weight?: number,
-    shouldNotify?: any
```

Comment:
@ballPointPenguin Not sure why `shouldNotify` has been removed here. Any knowledge of that?

Reply:
it is totally unused

Reply:
Hah! Just discovered this as well. Thanks for checking.